modules/apps/data-engine/data-engine-taglib/src/main/resources/META-INF/resources/data_layout_builder/js/components/sidebar/MultiPanelSidebar.es.js

```diff
@@ -161,76 +161,78 @@ export default function MultiPanelSidebar({panels, sidebarPanels}) {
 	return (
 		<ClayTooltipProvider>
 			<div className="multi-panel-sidebar">
-				<div className="multi-panel-sidebar__buttons">
-					{panels.reduce((elements, group, groupIndex) => {
-						const buttons = group.map(panelId => {
-							const panel = sidebarPanels[panelId];
+				<nav className="multi-panel-sidebar__buttons tbar tbar-light tbar-stacked">
```

Comment:
have you checked if the WEM wants the dark or light theme? If their designe want dark, we need to make it prop.

Reply:
Ok, I'll update the component to receive the tbar theme via props